src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    <Age age={person.age} />
+    <Partner
+      isMarried={person.isMarried}
+      sex={person.sex}
+      partnerName={person.partnerName}
+    />
+  </section>
+);
+
+const Age = ({ age }) => age && <p className="Person__age">{`I am ${age}`}</p>;
```

Comment:
Here's an alternative:

```suggestion
const Age = ({ age }) => {
  if (!age) {
    return null;
  }

return (
  <p className="Person__age">
    {`I am ${age}`}
  </p>
)};
```

I prefer this way more, as it looks more readable and clear to me. But again, depends heavily on the naming conventions of the project